ldk/javascript/src/webpack/generate-banner.ts

```diff
@@ -1,16 +1,33 @@
-export function generateMetadata(): string {
+import { LdkSettings } from './ldk-settings';
+
+export function generateMetadata(ldkSettings: LdkSettings): string {
+  if(Object.keys(ldkSettings.ldk).length === 0) {
+    throw new Error("Please provide LDK settings in your Loop package.json. See README for more information.")
```

Comment:
We could probably make this error message a bit more explicit about the expected remedy, maybe something like:

> Please add a "ldk" object to your package.json file with a permission property:
>
> ```
> "ldk": {
>   "permissions": {}
> }
> ```

Reply:
I accept this as well - we want to make sure folks know what to do when their transpilation fails.